# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/05_mm-2.7-fonts.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/fonts/Makefile.marc	Tue Feb 13 14:56:31 1996
+++ mm2.7/src/fonts/Makefile	Tue Feb 13 14:57:00 1996
@@ -28,11 +28,11 @@
 	-${DIRBUILDER}
 
 shownonascii: Xshownonascii
-	sed -e s@/u/nsb/lib/fonts@`pwd`@ < Xshownonascii > shownonascii
+	sed -e s@/u/nsb/lib/fonts@/usr/lib/metamail/fonts@ < Xshownonascii > shownonascii
 	chmod +x shownonascii
 
 mailto-hebrew: Xmailto-hebrew
-	sed -e s@/u/nsb/lib/fonts@`pwd`@ < Xmailto-hebrew > mailto-hebrew
+	sed -e s@/u/nsb/lib/fonts@/usr/lib/metamail/fonts@ < Xmailto-hebrew > mailto-hebrew
 	chmod +x mailto-hebrew
 
 clean:
